tests/unit/FormInput.spec.js

```diff
@@ -18,15 +24,15 @@ describe('FormInput', () => {
 
   it('should emit a value on mount', () => {
     const wrapper = factory();
-    expect(wrapper.emitted().input).toBeTruthy();
+    expect(wrapper.emitted()).toBeTruthy();
```

Comment:
Is there a specific value that should be emitted? Looking at the component, it's not obvious that a value should be emitted when the component mounts; it seems that functionality comes from the `formatFloatValue` function.

Reply:
Yes you are correct. I'm not sure if any component emits any value on mount.